src/components/ReviewConfirm/DocumentDelivery.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <v-card flat class="pr-5 pb-4">
```

Comment:
It's better for this component's parent to specify the whitespace around it.

Reply:
kk, will update

Reply:
Oh wait, this is padding within the component.

Reply:
Are you sure this isn't padding AROUND the component? Looks like the v-container on line 3 specifies padding within the component.

Reply:
Ah yea, there is another level of padding inside in the v-container. Maybe i can do away with the container. 
The card itself is the component tho

Reply:
Fixed up, in next commit!